js/sykmeldinger/sykmeldinger/AvvistSykmeldingTeaser.js

```diff
@@ -43,13 +74,47 @@ const AvvistSykmeldingTeaser = ({ smSykmelding }) => {
                     tittel={getLedetekst('sykmelding.teaser.tittel')}
                     id={id}
                     status="Avvist av NAV" />
+                <InngangspanelTekst>
```

Comment:
Syns dette bør testes med enhetstester...

Reply:
Jeg har laget tester for å finne rikgige ledetekstnøkler, tror du det holder?